CHANGELOG.md

```diff
@@ -1,6 +1,34 @@
 # Changelog
 All notable changes to the COBOL Language Support extension are documented in this file.
 
+## [0.17.0](https://github.com/eclipse/che-che4z-lsp-for-cobol/compare/0.16.1...0.17.0) (2021-02-01)
+
+### Bug Fixes
+* Fix semantic analysis stop after XML query ([d26b848](https://github.com/eclipse/che-che4z-lsp-for-cobol/commit/d26b8481111bb3ade6eb40f89e8282383583aea3)), closes [#706](https://github.com/eclipse/che-che4z-lsp-for-cobol/issues/706)
+* Fix semantic error on structure calls [#504](https://github.com/eclipse/che-che4z-lsp-for-cobol/issues/504) ([896ec1f](https://github.com/eclipse/che-che4z-lsp-for-cobol/commit/896ec1f558f4071e4cc46a3572274f779c867233))
+* Fix variable recognition fail on ambguity ([86b1423](https://github.com/eclipse/che-che4z-lsp-for-cobol/commit/86b1423107ec0cfc948d9fd6bf48fb24c3926b1e)), closes [#496](https://github.com/eclipse/che-che4z-lsp-for-cobol/issues/496)
+* Fix variable usages in conditions [#729](https://github.com/eclipse/che-che4z-lsp-for-cobol/issues/729) ([8f76b70](https://github.com/eclipse/che-che4z-lsp-for-cobol/commit/8f76b707c1047bc6b16fd19e4abd1d9e48af65dc))
+* MAPFAIL Produces a Semantic Error in HANDLE CONDITION ([2d778ea](https://github.com/eclipse/che-che4z-lsp-for-cobol/commit/2d778ea65737ae64ddf565baa00fd0829d34c456)), closes [#668](https://github.com/eclipse/che-che4z-lsp-for-cobol/issues/668)
```

Comment:
Missing "Fix" here

Reply:
Where? "Fix MAPFAIL produces a Semantic Error in HANDLE CONDITION" doesn't make any sense.